scripts/js/020-components/08-print.js

```diff
@@ -0,0 +1,23 @@
+// track print events in Google Analytics
+
+var printedCount = 0;
+
+function logPrintEvent() {
```

Comment:
This function name isn't very descriptive, it sends data to google analytics however it is called `log`.